recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -17,30 +22,32 @@ class UTConan(ConanFile):
     no_copy_source = True
     options = { "disable_module": [True, False], }
     default_options = { "disable_module": False, }
-    _cmake = None
 
     @property
     def _minimum_cpp_standard(self):
-        return 17 if self.settings.compiler in ["clang", "gcc"] and tools.Version(self.version) <= "1.1.8" else 20
+        return 17 if self.settings.compiler in ["clang", "gcc"] and Version(self.version) <= "1.1.8" else 20
 
     @property
     def _minimum_compilers_version(self):
         return {
-            "apple-clang": "11" if tools.Version(self.version) < "1.1.8" else "12",
+            "apple-clang": "11" if Version(self.version) < "1.1.8" else "12",
             "clang": "9",
             "gcc": "9",
             "msvc": "19",
             "Visual Studio": "16",
         }
 
+    def layout(self):
+        cmake_layout(self)
+
     def export_sources(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+            copy(self, patch["patch_file"], self.recipe_folder, self.export_sources_folder)
 
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
-        if tools.Version(self.version) <= "1.1.8" and self.settings.compiler in ["msvc", "Visual Studio"]:
+            check_min_cppstd(self, self._minimum_cpp_standard)
+        if Version(self.version) <= "1.1.8" and self.settings.compiler in ["msvc", "Visual Studio"]:
```

Comment:
```suggestion
        if Version(self.version) <= "1.1.8" and is_msvc(self):
```
from conan.tools.microsoft import is_msvc

Reply:
Done, thanks!